Tests/ValetIntegrationTests/MacTests.swift

```diff
@@ -97,6 +97,90 @@ class ValetMacTests: XCTestCase
         // This is not be the case upon setting a breakpoint and inspecting before the valet.setString(, forKey:) call above.
     }
 
+    func test_withExplicitlySet_identifierHasExplicitlySetIdentifier() {
+        let explicitlySetIdentifier = Identifier(nonEmpty: #function)!
+        Valet.permutations(withExplictlySet: explicitlySetIdentifier, shared: false).forEach {
+            XCTAssertEqual($0.keychainQuery?[kSecAttrService as String], explicitlySetIdentifier.description)
+        }
+
+        Valet.iCloudPermutations(withExplictlySet: explicitlySetIdentifier, shared: false).forEach {
+            XCTAssertEqual($0.keychainQuery?[kSecAttrService as String], explicitlySetIdentifier.description)
+        }
```

Comment:
I mean, this could be 4 tests. But I don't gain a lot by splitting this up?

Reply:
I'd indeed do 4 tests here 😁 

We mostly gain the ability that when a test fails, we immediately know what assert is causing it (since we'd have [1 per test](https://osherove.com/blog/2005/4/14/try-to-avoid-multiple-asserts-in-a-single-unit-test.html))

I'm a fan of that. I think using `XCTest` doesn't play that well with this though since you need to write a whole new test for that, but I also think this is fine though 🤷‍♂

Reply:
I think another solve for that is writing good descriptions for the failure case. In this case, I have 4 classes of tests, each with `Accessibility.allValues().count` asserts. That's... a lot of tests to write. And if we ever add a new accessibility value, it'll get worse.

Let me touch this up by adding some failure strings.

Fwiw, when one of these tests failed earlier in CI, I looked up the line number of the failure. That worked well enough 🤷‍♂